pull_request_template.md

```diff
@@ -19,18 +26,16 @@ Waiting on the following PRs:
 
 ## Impact
 
-If changes in this PR will affect other repositories, please add a "waiting for other repos" label, and list the repositories that will be affected to the best of your knowledge (example below).
-Requires changes in the following repositories:
-- [ ] saber
-- [ ] ioda
-- [ ] ufo
-- [ ] ...
-
-If changes in this PR require updating test data on AWS please add an "update test data" label and list the test data that needs to be updated to the best of your knowledge (example below).
-Requires updating AWS test data for the following repositories:
-- [ ] saber
-- [ ] ioda
-- [ ] ...
-
-Note: to automatically run saber, ioda and ufo tests with this PR, push a commit containing "trigger pipeline", e.g.:
-git commit --allow-empty -m 'trigger pipeline'
+Please list the other repositories (if any) that this PR will require changes in (example below)
+
+- saber
```

Comment:
I think it may be good to keep `- [ ]` syntax. In this case this shows up as a checklist, and one can see how many points are checked there from the Pull Requests general page, see e.g. in oops for some PRs: https://github.com/JCSDA-internal/oops/pulls.

Reply:
ok - I thought that was just one more thing to fill out, but good point.